library/oneview_storage_system.py

```diff
@@ -52,6 +52,12 @@
         description:
             - List with Storage System properties and its associated states.
         required: true
+    validate_etag:
+        description:
+            - When the ETag Validation is enabled, the request will be conditionally processed only if the current ETag
+              for the resource matches the ETag provided in the data.
+        default: true
+        choices: ['true', 'false']
```

Comment:
But the documentation doesn't have any mention about the type of this ansible argument, or the user can presume it?